src/components/Dashboard/Form/Applicants/Applicant.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import FieldWrapper from "../FieldWrapper/FieldWrapper";
+import styles from "./Applicant.scss";
+
+class Applicant extends Component {
+  state = {
+    isVisible: false
+  };
+  handleOnButton = () => {
+    if (this.state.isVisible === false) {
+      this.setState({ isVisible: true });
+    } else {
+      this.setState({ isVisible: false });
+    }
+  };
+  render() {
+    return (
+      <div className={styles.Applicant}>
+        <button
+          onClick={this.handleOnButton}
+          type="button"
+          className={styles.developedDiv}
+        >
+          {" "}
+          +{" "}
```

Comment:
What is this for?

Reply:
ESLint added this, but I corrected , as ESlint allows